documentcloud/documents/tasks.py

```diff
@@ -117,6 +119,23 @@ def redact(document_pk, slug, access, ocr_code, redactions):
     )
 
 
+@task(
```

Comment:
I added a helper function for all instances of HTTP calling out to lambda to make them more robust (was seeing errors here with large uploads) - this should be rebased onto `master` and adopt those changes